src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -242,6 +242,7 @@ def read_user_config(
     ):
         site_config = cls.read_site_config(use_new_parser=use_new_parser)
         if use_new_parser:
+            ConfigParser.check_non_utf_chars(user_config_file)
```

Comment:
I think we should handle this case differently for the new parser. We should be able to catch the error thrown by https://github.com/equinor/ert/blob/602860dd09f4e71db95f88fdc614a302b4dc13cf/src/ert/_c_wrappers/enkf/lark_parser.py#L512

and rethrow is as a `ConfigValidationError`.

Reply:
Sure why not